volatility3/framework/plugins/windows/pe_symbols.py

```diff
@@ -632,7 +633,7 @@ def _find_symbols_through_exports(
     def _get_symbol_value(
         wanted_symbols: filter_module_info,
         symbol_resolver: PESymbolFinder,
-    ) -> Generator[Tuple[str, int, str, int], None, None]:
+    ) -> Generator[Tuple[str, str, int], None, None]:
```

Comment:
The docstring needs changing too then...

Reply:
Nice catch.